test/WebJobs.Extensions.RabbitMQ.Tests/RabbitMQConfigurationTests.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ.Tests
+{
+    public class RabbitMQConfigurationTests
+    {
+        private static readonly IConfiguration _emptyConfig = new ConfigurationBuilder().Build();
+
+        [Fact]
+        public void Creates_Context_Correctly()
+        {
+            var options = new RabbitMQOptions { Hostname = "localhost", QueueName = "queue" };
+            var loggerFactory = new LoggerFactory();
+            var config = new RabbitMQExtensionConfigProvider(new OptionsWrapper<RabbitMQOptions>(options), (ILoggerFactory)loggerFactory);
+            var attribute = new RabbitMQAttribute { Hostname = "localhost", QueueName = "queue" };

```

Comment:
Set different values on options and attributes. And verify expected.
Also, add variations where options is null , attribute is null. Use [Theory]. See here for example: https://github.com/Azure/azure-webjobs-sdk-extensions/blob/4815a674e1e380d7d7e01d952f6c8fc107301be1/test/WebJobs.Extensions.CosmosDB.Tests/CosmosDBConnectionStringTests.cs#L11